Private/Find-ESC3C1.ps1

```diff
@@ -1,4 +1,4 @@
-﻿function Find-ESC3Condition1 {
+﻿function Find-ESC3C1 {
```

Comment:
IMHO, "ESC3Condition1" is more readable than "ESC3C1" to me. Others are welcomed to differ. 😊

Reply:
I agree with you! I was prepping for refactoring `Invoke-RiskRating` and thought this would make it easier. Maybe an alias would be a better option?

Reply:
Just to make sure I'm on the same page, is your intent for making `Invoke-RiskRating` easier to simply shorten the function name, or to standardize on a naming convention?

Reply:
Both. I think?